### DISCUSSION TITLE
What is actuation?

### DISCUSSION THREAD

**Discussion by Banzobotic** [pybricks/support#486] [Q&A]
What does the actuation setting in the control class limits() affect?

**Comment by laurensvalk** [comment]
It's the limit on the actuation signal of that controller. For a motor, this is the duty cycle. You can set a limit between 0 and 100%.

For drive bases, this setting is no longer used.

**Reply by Banzobotic** [reply]
I am using ev3. Is this like the percent power that you would see in the standard mindstorms software?
